core/src/main/java/org/moditect/commands/GenerateModuleInfo.java

```diff
@@ -372,10 +373,20 @@ private Map<String, Boolean> generateModuleInfo() throws AssertionError {
         command.add( inputJar.toString() );
 
         log.debug( "Running jdeps " + String.join(  " ", command ) );
-        int result = jdeps.run( System.out, System.err, command.toArray( new String[0] ) );
 
+        // Print error to a buffer stream
+        ByteArrayOutputStream buffer = new ByteArrayOutputStream();
+        PrintStream errorStream = new PrintStream(buffer);
+
+        int result = jdeps.run( System.out, errorStream, command.toArray( new String[0] ) );
```

Comment:
This won't help much, as it captures error stream output, which `jdeps` never actually uses:
```java
public class Main {
    public static int run(String[] args, PrintWriter out) {
        JdepsTask t = new JdepsTask();
        t.setLog(out);
        return t.run(args);
    }

    public static class JDepsToolProvider implements ToolProvider {
        public String name() {
            return "jdeps";
        }

        public int run(PrintWriter out, PrintWriter err, String... args) {
            return Main.run(args, out);
        }
    }
}
```
Basically, even with the change, everything is sent only to `System.out`, which I'm sure wasn't the goal.

Ideally, we shouldn't be capturing output here at all, but simply forwarding it to the `log`. That would ensure it works well both in standalone/CLI mode, and when run as a Maven plugin, where the log is replaced by `MojoLog`.

Something like this:
```java
public class LogWriter
        extends PrintWriter
    {
    private static final String ERROR_PREFIX   = "Error:";
    private static final String WARNING_PREFIX = "Warning:";

    private final Log log;

    /**
     * Creates a new PrintWriter that will write everything to the specified log
     * in addition to {@code System.out}.
     *
     * @param log the log to write to
     */
    public LogWriter(Log log)
        {
        super(System.out);

        this.log = log;
        }

    /**
     * Prints a String and then terminates the line.  This method behaves as
     * though it invokes {@link #print(String)} and then {@link #println()}.
     *
     * @param sText the {@code String} value to be printed
     */
    public void println(String sText)
        {
        super.println(sText);

        if (sText.startsWith(ERROR_PREFIX))
            {
            log.error(sText);
            }
        else if (sText.startsWith(WARNING_PREFIX))
            {
            log.warn(sText);
            }
        else
            {
            log.info(sText);
            }
        }
    }
```
which can then be used when running `jdeps`:
```
		LogWriter logWriter = new LogWriter(log);
		int result = jdeps.run( logWriter, logWriter, command.toArray( new String[0] ) );
```
Thoughts?